examples/gevent/gevent-server.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+"""
+gevent-server.py
+================
+
+A simple HTTP/2 server written for gevent serving static files from a directory specified as input.
+If no directory is provided, the current directory will be used.
+
+Requires Python 3.6+.
```

Comment:
FYI: hyper-h2 (with current git-master) now requires at least py36.

Reply:
I remove the requirement in the docstring, thanks!